dev-util/buildbot/buildbot-3.11.0.ebuild

```diff
@@ -0,0 +1,183 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_REQ_USE="sqlite"
+PYTHON_COMPAT=( python3_{10..12} )
+inherit readme.gentoo-r1 systemd distutils-r1 pypi
+
+DESCRIPTION="BuildBot build automation system"
+HOMEPAGE="https://buildbot.net/
+	https://github.com/buildbot/buildbot
+	https://pypi.org/project/buildbot/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~amd64-linux ~x86-linux"
+
+IUSE="crypt docker examples irc test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	acct-user/buildbot
+	~dev-util/buildbot-worker-${PV}[${PYTHON_USEDEP}]
+	>=dev-python/alembic-1.6.0[${PYTHON_USEDEP}]
+	>=dev-python/autobahn-0.16.9[${PYTHON_USEDEP}]
```

Comment:
Due to re-sorting in the same commit, changes are a *tiny* bit hard to follow.

autobahn is bumped to .9 here, but not upstream?